src/components/VSelect/VSelect.js

```diff
@@ -378,11 +380,12 @@ export default {
       this.isActive = true
       this.isFocused = true
 
-      if (this.$refs.input && this.isAutocomplete) {
-        this.$nextTick(() => {
-          // this.$refs.input.focus()
-        })
-      }
+      // TODO
```

Comment:
I think you can just go ahead and remove this.

Reply:
Moved it to my other PR instead